samples/nrf9160/multicell_location/prj.conf

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# General config
+CONFIG_NEWLIB_LIBC=y
+CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
```

Comment:
I've lately had trouble printing floats if CONFIG_FPU isn't explicitly enabled in the application configuration. You haven't had this issue?

Reply:
Added `CONFIG_FPU` as that will now allow printing of floats using the logger